docs/tutorial/part-two/index.md

```diff
@@ -114,7 +114,7 @@ import "./src/styles/global.css"
 // require('./src/styles/global.css')
 ```
 
-> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, we use `import` most of the time.
+> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, `import` will usually be the most common. When working with files that will only be run in the node env like `gatsby-node.js`, `require` will need to be used.
```

Comment:
I think this should be expanded on a bit more, mainly because to a new-comer to JS and also Gatsby might find it difficult to understand both the difference between import and require and also which environment `gatsby-browser.js` file  is used in. 

In reality it's both, which leads to more questions like, `how does import work here when I'm using a version node that doesn't support esm`.

Reply:
While this is valid concern, I'm not sure if diving into this topic in part 2 of tutorial is proper place for it?

Reply:
We might need more dedicate resource to explain this in depth (just not in the tutorial)

Reply:
Good point! This commons/esm topic here has a lot of other info around it and not the best place for it. Maybe a reference to another resource that would be helpful